### PR TITLE
bootstrap defaults

### PR DESCRIPTION
**Pull request by mandresm**

### CODE REVIEW

File: configs/setups/awiesm/awiesm-2.1.yaml [hunk 1]
```diff
@@ -161,8 +162,17 @@ echam:
                 OASIS3MCT_FC_LIB: "$(pwd)/lib/"
                 OASIS3MCTROOT: "$(pwd)/oasis/"
                 configure_opts: '--with-coupler=oasis3-mct'
-        add_test1:
-            test3: false
+
+        choose_general.with_bootstrap:
+                True:
+                        add_input_files:
+                                target_orography: target_orography
```
Comment (pgierz) [inline configs/setups/awiesm/awiesm-2.1.yaml:169]: What sort of error happens when this hasn't been added to `add_input_sources`?

**Review by pgierz — COMMENTED** [review]
Looks good, just one question about errors which we might need to think about presenting in a clear way.

File: configs/setups/awiesm/awiesm-2.1.yaml (hunk 1 above)
Comment (mandresm) [reply configs/setups/awiesm/awiesm-2.1.yaml:169]: This error:
```
First year, checking if we need to compile...

================================================================================
STARTING SIMULATION JOB!
Experiment ID = bootstrap
Setup = awiesm
This setup consists of: 
- echam
- fesom
- oasis3mct
- jsbach
- hdmodel
Experiment is installed in: 
       /work/ollie/mandresm/testing//run/awiesm//bootstrap
================================================================================

Implementation target_orography not found for filetype input of model echam
{'cldoptprops': 'cldoptprops', 'janspec': 'janspec', 'jansurf': 'jansurf', 'rrtmglw': 'rrtmglw', 'rrtmgsw': 'rrtmgsw', 'tslclim': 'tslclim', 'vgratclim': 'vgratclim', 'vltclim': 'vltclim', 'target_orography': 'target_orography'}
{'cldoptprops': '/work/ollie/pool/ECHAM6//input/r0007/ECHAM6_CldOptProps.nc', 'janspec': '/work/ollie/pool/ECHAM6//input/r0007/T63/T63L47_jan_spec.nc', 'jansurf': '/work/ollie/pool/FESOM/meshes_default/core/tarfilesT63/input/echam6/T63CORE2_jan_surf.nc', 'rrtmglw': '/work/ollie/pool/ECHAM6//input/r0007/rrtmg_lw.nc', 'rrtmgsw': '/work/ollie/pool/ECHAM6//input/r0007/rrtmg_sw.nc', 'tslclim': '/work/ollie/pool/ECHAM6//input/r0007/T63/T63_TSLCLIM.nc', 'vgratclim': '/work/ollie/pool/FESOM/meshes_default/core/tarfilesT63/input/echam6/T63CORE2_VGRATCLIM.nc', 'vltclim': '/work/ollie/pool/FESOM/meshes_default/core/tarfilesT63/input/echam6/T63CORE2_VLTCLIM.nc'}
2022-03-30 16:53:44.313268
Copying standard yamls from:  /home/ollie/mandresm/esm_tools/configs/./
Copying standard namelists from:  /home/ollie/mandresm/esm_tools/namelists/./
/home/ollie/mandresm/esm_tools/src/esm_tests/resources/runscripts/awiesm/bootstrap.yaml
```

Comment (pgierz) [reply configs/setups/awiesm/awiesm-2.1.yaml:169]: Could be prettier, but if we document it cleanly, that's good enough for me!

**Pull request merged**